### DISCUSSION TITLE
Notifications for discussions thread

### DISCUSSION THREAD

**Discussion by driusan** [community/community#3002] [Discussions]
Is there any way to get GitHub notifications when a thread is created or replied to? I don't seem to be getting any, and as far as I can tell I need to re-visit the discussions tab to see if anything's new.

**Comment by driusan** [comment]
I seem to have somehow just missed the notifications, they're there in my read notifications.. but I can't seem to close/delete this, even as the author.

**Reply by becca** [reply]
Glad you were able to figure things out! We are aware of the request/need to delete a discussion you have authored as it was mentioned over in https://github.com/github/feedback/discussions/2948 but right now we only allow for repo owners to delete Issues and Discussions within a repository.
